lib/message/predictions.ex

```diff
@@ -30,12 +30,13 @@ defmodule Message.Predictions do
        )}
     end
 
+    # Show 4-car messages at non-terminal Red Line stops with the exception of Ashmont
     def to_multi_line(
           %Message.Predictions{
-            terminal?: false,
             predictions: [%{route_id: "Red", multi_carriage_details: [_, _, _, _]} = top | _]
           } = message
-        ) do
+        )
+        when (message.terminal? and top.stop_id == "70094") or not message.terminal? do
```

Comment:
We should be able to drop `message.terminal?` from `(message.terminal? and top.stop_id == "70094")`. We won't be showing a prediction for `70094` (RL Ashmont NB platform) anywhere except for at that platform and at the mezzanine